packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -113,6 +113,7 @@ const useLiveChat = (has_cookie_account = false, active_loginid?: string) => {
         history.listen(onHistoryChange);
 
         window.LiveChatWidget?.on('ready', () => setIsReady(true));
+        return () => setIsReady(false);
```

Comment:
this cleanup function seems to fix a console error being displayed.

Reply:
we're not supposed to set state on unmount I suppose, the better approach can be to utilize `useIsMounted` and refactor it a bit to not setting a status upon cleanup

Reply:
The memory leak error from react is related to when we try to update the state in a component which is unmounted

Reply:
didn't knew about `useIsMounted`. it worked well 👌🏼 
thanks man ❤️